.travis.yml

```diff
@@ -6,8 +6,9 @@ cache: pip
 before_install:
   - sudo apt-get update
 install:
-  - pip install setuptools --upgrade
+  - pip install setuptools==37
```

Comment:
May I know the reason why we should specify the version?

Reply:
See my PR description.